lib/adobe-commerce.js

```diff
@@ -173,65 +173,65 @@ async function getAdobeCommerceClient(params) {
     // Commerce Rest API: https://developer.adobe.com/commerce/webapi/rest/quick-reference/
     createOopePaymentMethod: async (paymentMethod) =>
       wrapper(() =>
-        commerceGot(`rest/all/V1/oope_payment_method/`, {
+        commerceGot(`V1/oope_payment_method/`, {
           method: 'POST',
           json: paymentMethod,
         }).json()
       ),
     getOopePaymentMethod: async (paymentMethodCode) =>
       wrapper(() =>
-        commerceGot(`rest/all/V1/oope_payment_method/${paymentMethodCode}`, {
+        commerceGot(`V1/oope_payment_method/${paymentMethodCode}`, {
```

Comment:
And this is not going to work with PAAS right?

Reply:
It works with PaaS, but `COMMERCE_BASE_URL` won't be anymore instance URL but base REST endpoint URL. Is this example in the comment clear enough for you? 

https://github.com/adobe/commerce-checkout-starter-kit/blob/7a7215be8c4443edd015cca3c2cf054f2ddebbd9/env.dist#L28

Please note that we also need to update the document later on: https://developer.adobe.com/commerce/extensibility/starter-kit/checkout/connect/
```
To use the Adobe Commerce HTTP Client, update COMMERCE_BASE_URL=<commerce_instance_url> in the .env file, and complete the authentication setup.
```